test/Tensor_test.jl

```diff
@@ -41,6 +41,22 @@
 
         @test tensor ∈ Set([tensor])
         @test zeros(size(tensor)...) ∉ Set([tensor])
+
+        @test tensor == permutedims(tensor, (3, 1, 2))
+    end
+
+    @testset "isapprox" begin
+        data = rand(2, 2, 2, 2)
```

Comment:
Try different sizes.
```suggestion
        data = rand(2, 3, 4, 5)
```